rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionInterpreter.java

```diff
@@ -45,9 +46,20 @@ public SolutionInterpreter( Solution solution, boolean appendWon )
         this.command = null;
         this.untilCount = -1;
     }
+    
+    public static SolutionInterpreter getNothingPlaying()
+    {
+        SolutionInterpreter nothingPlaying = new SolutionInterpreter( SolutionParser.parse( "1" ) );
+        nothingPlaying.emptySteps = true;
+        return nothingPlaying;
+    }
```

Comment:
Better to pass in emptySteps in the constructor and provide a constructor that defaults it. Then it (emptySteps) can be final. Also can we think of a more descriptive name? Continue past end?